### PR TITLE
Remove remaining references to `SequentialTaskRunner` in 3.x docs

### LINKED ISSUE #13955: Remove references to `SequentialTaskRunner` in 3.0 docs
### First check

- [X] I added a descriptive title to this issue.
- [X] I used GitHub search to find a similar request and didn't find it 😇

### Describe the issue

`SequentialTaskRunner` no longer exists

### Describe the proposed change

Remove references from the docs

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by zzstoatzz**
closes #13955

note we keep one reference to it in the task runners page to let people know that its gone now (should they come looking for it)

### CODE REVIEW

File: docs/2.19.x/concepts/flows.mdx [hunk 1]
```diff
@@ -120,9 +120,10 @@ For example, you can provide a `name` value for the flow. Here we've also used t
 from prefect import flow
 from prefect.task_runners import SequentialTaskRunner
 
-@flow(name="My Flow",
-      description="My flow using SequentialTaskRunner",
-      task_runner=SequentialTaskRunner())
+@flow(
+    name="My Flow",
+    description="My flow using SequentialTaskRunner",
```
Comment (zzstoatzz) [inline docs/2.19.x/concepts/flows.mdx:125]:
```suggestion
    description="My flow using SequentialTaskRunner",
    task_runner=SequentialTaskRunner())
```

**Review by chrisguidry — APPROVED** [review]
Noice!

**Review by billpalombi — APPROVED** [review]
Thank you!

File: docs/2.19.x/concepts/flows.mdx [hunk 2]
```diff
@@ -120,9 +120,11 @@ For example, you can provide a `name` value for the flow. Here we've also used t
...
+    task_runner=SequentialTaskRunner())
```
Comment (desertaxle) [inline docs/2.19.x/concepts/flows.mdx:126]: Why are we keeping the `SequentialTaskRunner` in this example? This example doesn't seem specific to the `SequentialTaskRunner`

Comment (zzstoatzz) [reply docs/2.19.x/concepts/flows.mdx:126]: this is 2.19.x, its showing how to use settings

**Pull request merged**